src/components/ui/footer.tsx

```
@@ -0,0 +1,56 @@
+                    <p className='text-sm text-justify pt-8 mb-8'>Our platform brings you the latest merchandise from the College of Computer Studies. Whether you're looking for shirts, hoodies, or accessories, easily browse, order, and rep your college spirit in style!</p>
+                </div>
+                <div className='flex flex-col min-w-max max-w-2xl w-full'>
+                    <h1 className='font-bold text-7xl 2xl:self-start self-center'>Get in Touch</h1>
+                    <h1 className='text-4xl py-8 2xl:self-start self-center'>tminusone@gmail.com</h1>
+                </div>
+            </div>
+            
+            <div className='flex flex-col md:flex-row flex-wrap justify-between mx-12 pb-4 px-12'>
+                <hr className='place-self-center w-full mt-4 pb-4'/>
+                <div className='flex flex-col xl:flex-row justify-between w-full items-center text-center xl:text-left'>
+                    <span className="block text-base font-bold sm:text-center pb-2">© 2023 <a href="#" className="hover:underline">MerchTrack™</a>. All Rights Reserved.</span>
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)